crates/bevy_transform/src/systems.rs

```diff
@@ -320,7 +320,7 @@ mod test {
 
         // Adding the system in a single threaded stage. As the system will panic, this will
```

Comment:
is this comment outdated? It would appear that this is adding the system to a multi-threaded stage.